### PR TITLE
Take into account Imperialism

### PR DESCRIPTION
**Pull request by TannerBaldus**
Added Guam and Puerto Rico. I couldn't find anything on American Samoa, Virgin Islands, or the Northern Mariana Islands.
sourced from http://www.corragroup.com/puerto-rico-motor-vehicle-records.html and http://www.corragroup.com/guam-driving-records.html.

### CODE REVIEW

File: regex.json [hunk 1]
```diff
@@ -268,6 +274,12 @@
       "8 Numeric"
     ]
   },
+  "PR": {
+    "rule": "(^[0-9]{9}$)|(^[0-9]{5,7}$)",
+    "description": [
+      "5-7 Numeric or 9 Numeric"
```
Comment (adambullmer) [inline regex.json]: Can you split this into 2 array elements?

**Comment by adambullmer** [comment]
Thanks for adding these! I had just a small comment on the Puerto Rico descriptions, otherwise great work!

**Comment by TannerBaldus** [comment]
No problem! Fixed.

**Comment by adambullmer** [comment]
Thanks man, I'll add those unit tests, since those were added after you had initially created this PR

**Pull request merged**